integration/airflow/openlineage/airflow/adapter.py

```diff
@@ -91,8 +93,10 @@ def build_task_instance_run_id(task_id, execution_date, try_number):
     def emit(self, event: RunEvent):
         event = redact_with_exclusions(event)
         try:
-            return self.client.emit(event)
+            with Stats.timer("ol.emit.attempts"):
```

Comment:
This is a great improvement, @harels ! Since we care about this feature, would it make sense to extend existing tests or add a test to confirm we're covering sending metrics to stats?